### PR TITLE
Added use cases

### PR DESCRIPTION
**Pull request by PieterKas**
Added uses cases to the appendix (see issue #68 )

### CODE REVIEW

**Comment by PieterKas** [comment]
@aaronpk please review this PR - I added the use cases relevant to your draft.

File: draft-ietf-oauth-identity-chaining.md [hunk 1]
```diff
@@ -294,6 +294,12 @@ A continuous integration system needs to access external resources, for example
...
 
+## Extend Single-Sign-On to API Access
+A user that authenticated to an enterprise Identity Provider (IdP) do not have to sign-in to multiple SaaS applications if the SaaS applications are configured to trust the enteprise IdP. This SSO relationship is extended to API access. As a result SaaS servers that trust the enterprise IdP do not require the user to complete an interactive delegated OAuth 2.0 flow to obtain an access token to access the SaaS provider's APIs.
+
+## Cross-domain API authorization
+An e-mail client can be used with arbitrary email servers, without require pre-established relationships between each email client and each email server. When the e-mail client needs access to a separate API, such as a third-party calendaring application, the email client obtain an access token for the third-party calendaring application without any user interaction.
```
Comment (bc-pi) [inline draft-ietf-oauth-identity-chaining.md:301]: I must admit that it's not obvious to me how the use-case described is facilitated by the functionality in the document.

Comment (PieterKas) [reply draft-ietf-oauth-identity-chaining.md:301]: Yeah. I went back and forth about how much the use case had to be focused on the scenarios as opposed to describing how the mechanism enables it. I decided to be consistent with the other use cases (which actually pre-dated the spec).

Perhaps this section needs to be expanded to explain how the spec address the use cases as well (and that needs to be done for all the use cases I think). 

What do you think?

Comment (bc-pi) [reply draft-ietf-oauth-identity-chaining.md:301]: To be honest, my comment was made in the context of reading/reviewing this PR in isolation. I could not clearly make the mental connection myself between the described use case and the details of the spec. I don't recall encountering the same thing previously when reading the other use cases.

**Comment by PieterKas** [comment]
@aaronpk, @bc-pi I added some additional details to the use cases to illustrate the use cases Aaron describes in his identity assertion authz draft.

**Review by kburgin3 — APPROVED** [review]

**Comment by PieterKas** [comment]
@arndt-s can you merge this pull request? I don't have write permissions.

**Pull request merged**